internal/aiptest/update/required_fields.go

```diff
@@ -28,7 +28,7 @@ var requiredFields = suite.Test{
 	),
 	Generate: func(f *protogen.GeneratedFile, scope suite.Scope) error {
 		updateMethod, _ := util.StandardMethod(scope.Service, scope.Resource, aipreflect.MethodTypeUpdate)
-		util.RangeRequiredFields(scope.Message.Desc, func(p protopath.Path, field protoreflect.FieldDescriptor) {
+		util.RangeRequiredFields(scope.Message.Desc, true, func(p protopath.Path, field protoreflect.FieldDescriptor) {
```

Comment:
Lets use `xrange.RangeFields` directly (`util.RangeRequiredFields` is just a wrapper around it), and then filter do the filtering in this method instead. "Naked" boolean arguments are hard to reason about in review..

Reply:
I opted for just checking the annotation here and ignoring it if present. WDYT? Not sure if we'd like to limit annotation checking to certain packages or similar, so feedback welcome again 🙏

Reply:
I like it!